site/src/pages/components/switch.explainer.mdx

```diff
@@ -525,9 +525,16 @@ The switch element can be associated with the existing `role="switch"`.
 ### Globalization
 
 <div class="question">
-  Is thumb in the "on" position on the left in a `rtl` directional environment?
+  How should the switch behave in writing modes other than `horizontal-tb`?
```

Comment:
Since you have an answer to the question, please remove the question

Reply:
That said, we didn't resolve on the vertical scenario so there should probably be a new question elsewhere (maybe replace this text) and open a new issue and link to that one.